stores/sql/database.go

```diff
@@ -146,4 +167,16 @@ type (
 		FCID        FileContractID
 		RenewedFrom FileContractID
 	}
+
+	ContractInfo struct {
+		RevisionHeight uint64
+		RevisionNumber string
+		Size           uint64
+	}
 )
+
+func (c ContractInfo) RevNumber() uint64 {
+	var r uint64
+	fmt.Sscan(c.RevisionNumber, &r)
```

Comment:
I'd introduce a `Uint64` type in `/stores/sql/types.go` which implicitly converts the number from a string when scanning.